GoogleForCreators/web-stories-wp
# Global Page Advancement: use global values when creating templates

## Context
Follow-up for #12460 where it was decided that templates should also use global value.


## Summary



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Go to Settings and set Global Page Advancement values.
2. Now go to Explore templates and choose any template, click on Use template
3. When a new story is created with the template, go to the Document tab and verify the globally set page advancement values are used there.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Follow-up for #12460
